packages/webapp/src/components/forms/AddRequestForm/index.tsx

```diff
@@ -143,29 +133,69 @@ export const AddRequestForm: StandardFC<AddRequestFormProps> = wrap(function Add
 										/>
 									</Col>
 								</Row>
-								<Row className='flex-column flex-md-row mb-0'>
-									<Col className='mb-3 mb-md-0'>
-										<FormSectionTitle>{t('addRequestFields.addDuration')}</FormSectionTitle>
-									</Col>
-								</Row>
 								<Row className='flex-column flex-md-row mb-4'>
-									<Col className='mb-3 mb-md-0'>
-										<FormikField
-											name='duration'
-											type='number'
-											min={1}
-											max={999}
-											placeholder={t('addRequestFields.addDurationPlaceholder')}
-											className={cx(styles.field, 'duration')}
-										/>
-									</Col>
-									<Col className='mb-3 mb-md-0'>
-										<FormikSelect
-											name='durationUnit'
-											className='requestDurationSelect'
-											placeholder={t('addRequestFields.addDurationUnitPlaceholder')}
-											options={durationUnits}
-											defaultValue={durationUnits[0]}
+									<Col>
+										<FormSectionTitle>
+											<>
+												{t('addRequestFields.addEndDate')}{' '}
+												<span className='text-normal'>({t('addRequestFields.optional')})</span>
+											</>
+										</FormSectionTitle>
+
+										<DatePicker
+											placeholder={t('addRequestFields.addEndDatePlaceholder')}
+											allowTextInput
+											showMonthPickerAsOverlay={false}
+											ariaLabel={t('formElements.datePickerAriaLabel')}
+											value={values.endDate ? new Date(values.endDate) : null}
+											onSelectDate={(date) => {
+												setFieldValue('endDate', date)
+											}}
+											formatDate={(date) => date.toLocaleDateString(locale)}
+											minDate={new Date()}
+											styles={{
```

Comment:
Can you move the style object to the top of the page, to be more consistent with the rest of the codebase?